src/Core/src/Hosting/TypeConversionAppBuilderExtensions.cs

```diff
@@ -0,0 +1,58 @@
+#nullable enable
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.Maui.Hosting
+{
+	public static class TypeConversionAppBuilderExtensions
+	{
+		private static Dictionary<Type, TypeConverter>? s_typeConverters;
```

Comment:
Not sure if we can help it with various callers, but if this is static, the tests will have issues because multiple builder tests run in parallel - and each creates a new builder.

Is there a way to put this list into the service collection?

Reply:
I was trying to figure this out but in the end I just gave up. The alternative I was considering was registering it as a service and then accessing it from the static `TypeConversion` class via `Application.Current?.FindMauiContext()` which we seem to often use for logging. I'm not sure if that solves the problem though.